src/activities/send-version-to-epp.ts

```diff
@@ -0,0 +1,57 @@
+import axios from 'axios';
+import {
+  BlockContent, CreativeWorkTypes, InlineContent, Node, Organization, Person,
+} from '@stencila/schema';
+import { config } from '../config';
+import { EPPPeerReview } from './fetch-review-content';
+
+export type Content = Array<Node> | Node;
+
+export type Heading = {
+  id: string,
+  text: Content,
+};
+
+export type EnhancedArticle = {
+  id: string,
+  msid: string,
+  doi: string,
+  versionIdentifier: string,
+  versionDoi?: string,
+  // When we drop the old article schema from the DB,
+  // we can change ProcessedArticle to exclude these properties and drop `Omit` here
+  article: {
+    title: string | InlineContent[] | undefined,
+    authors: (Organization | Person)[] | undefined,
+    abstract: string | BlockContent[] | InlineContent[] | undefined,
+    licenses: (string | CreativeWorkTypes)[] | undefined,
+    content: (string & BlockContent[]) | (Node[] & BlockContent[]) | undefined,
+    headings: Heading[],
+    references: (string | CreativeWorkTypes)[] | undefined,
+  },
+  preprintDoi: string,
+  preprintUrl: string,
+  preprintPosted: Date,
+  sentForReview?: Date,
+  peerReview?: EPPPeerReview,
+  published?: Date,
+};
+
+type EPPImportResponse = {
+  result: boolean,
+  message: string,
+};
+
+export const sendVersionToEpp = async (versionJSON: EnhancedArticle): Promise<boolean> => {
+  const versionImportUri = `${config.eppServerUri}/import-version`;
```

Comment:
This endpoint isn't working for me. I know it isn't in the master branch of server yet. I can't see a corresponding PR on server. We should aim to get this endpoint in server ahead of this.